pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -24,13 +25,19 @@ stdenv.mkDerivation {
 
   buildInputs = [
     bison flex perl pkgconfig libpcap lua5 openssl libgcrypt gnutls
-    geoip libnl c-ares python libcap glib
+    geoip libnl c-ares python libcap glib zlib
```

Comment:
good question, the qt variant needs it, did not check gtk variant. Will get back to you on that.